packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,100 @@
+
+For example, a user could start an authentication flow by entering only a username, and this flow would prompt the client to request more information, or remediation, as required. Remediation is the direct communication between the client and the Identity Engine that can be achieved either without a browser redirect or with a browser redirect when the remediation transitions through a push authorization request. An example of a remediation step is the client prompting the user for a password or to add a second factor and then sending that information directly to the Identity Engine component of the Okta org for verification. Each form of remediation that the user must supply is dictated by the evaluated policies between the Okta org and the app. When the remediation steps are completed, the Identity Engine provides the client with an Interaction Code that is then used to exchange for tokens by an [Okta Authentication Server](/docs/concepts/auth-servers/), following [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid) standards.
+
+The Interaction Code grant is intended for developers who want a step-by-step remediation user experience without redirecting to an authorization server. This grant type enables developers to include [Identity Engine features](https://help.okta.com/okta_help.htm?type=oie&id=ext-features) in their app, such as passwordless authentication and progressive profiling. The Interaction Code flow can use policy-driven remediation steps not only for authentication, but also for registration and account recovery. See [Redirect authentication vs. embedded authentication](/docs/concepts/redirect-vs-embedded/) for Identity Engine authentication deployment models and [Identity Engine deployment guides](/docs/guides/oie-intro/) for detailed deployment use cases.
+
+## The Interaction Code flow
+
+The Interaction Code flow is similar to the [OAuth 2.0 Authorization Code flow with PKCE](/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce). All clients are required to pass along a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. Confidential clients such as web apps must also pass a client secret in their authorization request. The user can start the authorization request with minimal information, relying on the client to facilitate the interactions with the Okta org to progressively authenticate the user. The series of interactions, which could include multifactor authentication steps, is secured using the `interaction_handle`. After successfully completing the remedial interactions, the client receives an `interaction_code` that they can then redeem for tokens from an Okta Authorization Server with the [`/token`](/docs/reference/api/oidc/#token) endpoint.
```

Comment:
"is secured using the interaction_handle" -> when I read this, I thought "how?" I doubt we need a huge deep explanation of this, but it'd be good to add a couple of sentence to say what an interaction_handle is, and how it is used to secure the interaction flow.

Reply:
I removed "secured" and explained that the interaction_handle acts as a state handle for each transaction. Thanks

Reply:
nice one